sql/sql_union.cc

```diff
@@ -2435,10 +2435,8 @@ bool st_select_lex_unit::exec()
 
       fake_select_lex->table_list.empty();
       if (likely(!saved_error))
```

Comment:
So we end up with
```cpp
      if (likely(!saved_error))
        thd->limit_found_rows=
            (ulonglong) table->file->stats.records + add_rows;
```
coding style somewhere says one needs to use { } if the body of if/while/etc statement takes multiple lines, so use
```cpp
      if (likely(!saved_error))
      {
        thd->limit_found_rows=
            (ulonglong) table->file->stats.records + add_rows;
      }
```